otherlibs/stdune/src/char.ml

```diff
@@ -7,3 +7,5 @@ let is_digit = function
 let is_lowercase_hex = function
   | '0' .. '9' | 'a' .. 'f' -> true
   | _non_lowercase_hex_char -> false
+
+let hash = code
```

Comment:
We cared about uniformity for `int`, shouldn't we also pass this through `Int.hash`?

Reply:
Agreed.

Reply:
I changed it use `Int.hash` as well.

Might be worth asking this at base.

Reply:
Indeed. Thanks!